modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -101,7 +101,7 @@ def self.get_partname(headers)
     def self.get_content_type(headers)
       headers.each do |header|
         name, _, value = header.partition(':')
-        return value.split(';')[0].strip if name == 'Content-Type'
+        return value.split(';')[0].strip if name.downcase == 'content-type'
```

Comment:
rack-test start sending headers downcased, so I just made these comparisons agnostic of the case of the header

Reply:
related: https://github.com/rack/rack/issues/1592